src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+}
+
+.image {
+  height: 40px;
+  width: 40px;
+  align-self: center;
+  margin-left: 50px;
+  flex-grow: 1;
+}
+
+.nav {
+  height: 60px;
+  position: relative;
+  display: inherit;
+  right: 50px;
+}
+
+.nav--list {
+  display: inherit;
+  text-decoration: none;
+  text-transform: uppercase;
+  list-style-type: none;
+  padding: 0;
+  flex-grow: 1;
+  justify-content: flex-end;
+  margin: 0;
+}
+
+.nav--item {
+  margin-left: 20px;
+  display: inherit;
```

Comment:
do you need it?

Reply:
yes, if i dont use it. the result will be: 
![image](https://user-images.githubusercontent.com/90213755/225898395-3d46d0c4-54e0-4426-babc-a5c3e4728955.png)